python/ccxt/async_support/base/exchange.py

```diff
@@ -180,7 +180,7 @@ async def load_markets_helper(self, reload=False, params={}):
                     return self.set_markets(self.markets)
                 return self.markets
         currencies = None
-        if self.has['fetchCurrencies']:
+        if self.has['fetchCurrencies'] == true:
```

Comment:
Check these linting issues: https://app.travis-ci.com/github/ccxt/ccxt/builds/243221395#L1272

It has to be capitalized boolean `True` in Python.